include/dsl/idstring.hh

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <string>
+
+namespace dsl {
+	typedef uint64_t idstring;
```

Comment:
Are we confident then that idstring is not a class or structure who's first member is an integer? My only reason for being concerned is for memory compatibility reasons and for future proofing.

Reply:
Good point - I was only thinking about reading values.